### ISSUE TITLE
UX - Basic Onboarding

### ISSUE THREAD

**Issue by iamjoshfrank** [revelry-foundation/nightingale#58]

**Comment by HauwaAguillard** [comment]
Onboarding copy from @jwietelmann 
>These are my bullet points, and I think they're all pretty important and may not be readably displayable all on one screen?
>- When you start using Nightingale, the app tracks your location and saves that data on your device only.
>- Nightingale queries our servers to cross-check if you may have come into contact with someone who reported that they were COVID-19 positive. The server will not receive any identifying information about you or save any location information.
>- After you have been tracking your locations locally, you may CHOOSE to submit an anonymous report indicating that you are or were COVID-19 positive. This helps other users discover if they may have been near you while you were contagious. It is completely optional to do so, and doing so is completely anonymous.
>- We would never ask you to simply trust us with this amount of privacy. That's why the complete source code for every piece of the app and its infrastructure are available on [github](link)

See comment [in slack](https://revelry-public.slack.com/archives/C014AR56T1P/p1591303975075300?thread_ts=1591299696.063100&cid=C014AR56T1P)

**Comment by jwietelmann** [comment]
If any of my bullets can be shortened while still conveying the info, great please do that. Also (maybe this is obvious) the way it's displayed should not feel like a wall of text or anything that makes people's eyes glaze over or makes it feel similar to a ToS. Unlike a ToS, I think we _want_ them to _read every word_.

**Comment by HauwaAguillard** [comment]
Yeah, that makes sense. I think utilizing some icon set here will be helpful too

**Comment by HauwaAguillard** [comment]
**EOD:** 
Paired with @Samkeer1 and @BenRongey on the initial flow setup. We plan to continue progress on this ticket tomorrow. 

@Samkeer1 already pushed up a working branch

**Comment by iamjoshfrank** [comment]
My goal is to loop back to this tomorrow to provide updated content.

**Comment by HauwaAguillard** [comment]
While we ( myself & @Samkeer1 )wait for content from @iamjoshfrank, in the meantime, we're going to use the content provided by Joel on the onboarding screens. 

>- When you start using Nightingale, the app tracks your location and saves that data on your device only.
>- Nightingale queries our servers to cross-check if you may have come into contact with someone who reported that they were COVID-19 positive. The server will not receive any identifying information about you or save any location information.
>- After you have been tracking your locations locally, you may CHOOSE to submit an anonymous report indicating that you are or were COVID-19 positive. This helps other users discover if they may have been near you while you were contagious. It is completely optional to do so, and doing so is completely anonymous.
>- We would never ask you to simply trust us with this amount of privacy. That's why the complete source code for every piece of the app and its infrastructure are available on [github](link)

**Comment by iamjoshfrank** [comment]
I do think the content will need updating to be better suited to the end users, but I haven't been able to redirect my focus to this Issue yet today.

I'll keep an eye on the PRs and if I have time this weekend, I may open a PR with strictly content changes. If not, then picking it back up Monday pending all priorities.

**Comment by Samkeer1** [comment]
kicked back because https://github.com/revelry-foundation/nightingale-mobile/pull/27#issuecomment-640758447 wasn't addressed
